src/layouts/MainLayout/BottomNav.js

```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
+import React,{useEffect} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import {HomeOutlined, PlayCircleOutline, AccountCircleOutlined, AddCircleOutlineOutlined} from '@material-ui/icons';
+import { useHistory, useLocation } from 'react-router';
+=======
```

Comment:
same

Reply:
Thanks for the video you shared. Well I am learning loads.🙂